### PR TITLE
feat: shell script to run all the commands to start the processor

### PR DESCRIPTION
**Pull request by jtfirek**
I find it useful when testing.

### CODE REVIEW

File: runProcessor.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+nvm use
+yarn install
+yarn build
+docker-compose up -d db-kintsugi-testnet
+yarn db:migrate
```
Comment (bvotteler) [inline runProcessor.sh:7]: Does this work if the container has been stopped before?
I'm wondering if it needs to wait for docker to spin up the db before attempting `db:migrate`

You could eg. start the db first before doing `nvm use` through to `yarn build` . That might buy enough time more reliably before trying to run the migration.

Comment (bvotteler) [reply runProcessor.sh:7]: Or find a way to wait for the container to have been started before continuing - which would be the best way to make sure we can run migrate.

Comment (jtfirek) [reply runProcessor.sh:7]: Ahhh switching the order is clever. I will make that change.

Comment (jtfirek) [reply runProcessor.sh:29]: I made the change you requested and added a fun box to allow users to choose their network!

Comment (bvotteler) [reply runProcessor.sh:29]: > I made the change you requested and added a fun box to allow users to choose their network!

I like the idea. But, during startup the .env file dictates which network the processor is expecting to be run.
Can we use that to our advantage in this script? Something to think about.

Comment (jtfirek) [reply runProcessor.sh:64]: Great idea! I adding functionality to script to automatically set the environment variables based on the chosen network.

File: .env [hunk 2]
```diff
@@ -30,11 +30,11 @@
 #DB_PORT=5436
 
 #Kintsugi (testnet)
-ARCHIVE_ENDPOINT=https://api-dev-kintsugi.interlay.io/subsquid-gateway/graphql
-CHAIN_ENDPOINT=wss://api-dev-kintsugi.interlay.io/parachain
-SS58_CODEC="kintsugi"
-BITCOIN_NETWORK='testnet'
-DB_PORT=5433
+# ARCHIVE_ENDPOINT=https://api-dev-kintsugi.interlay.io/subsquid-gateway/graphql
```
Comment (bvotteler) [inline .env:33]: Please restore `.env` to its original state. There are other processes that rely on the values being set.

File: runProcessor.sh (hunk 1 above)
Comment (bvotteler) [reply runProcessor.sh:64]: > Great idea! I adding functionality to script to automatically set the environment variables based on the chosen network.

I meant the other way around:
Read the env variables and then the script selects the db to start based on what has been set, eg. in ARCHIVE_ENDPOINT

**Review by bvotteler — CHANGES_REQUESTED** [review]

File: runProcessor.sh (hunk 1 above)
Comment (jtfirek) [reply runProcessor.sh:35]: Ah I see what you mean. I changed the functionality to work the other way around

File: .env (hunk 2 above)
Comment (jtfirek) [reply .env]: I fixed this.

**Comment by jtfirek** [comment]
I removed the nvm check in this branch like you asked in pp #77 @bvotteler

**Review by bvotteler — APPROVED** [review]
Good stuff, thank you!

**Pull request merged**